src/firefly/js/visualize/ui/FitsHeaderView.jsx

```diff
@@ -0,0 +1,183 @@
+/*
+ L. Zhang Initial version 3/16/16
+ DM-4494:FITS Visualizer porting: Show FITS Header
+ */
+
+import React, {PropTypes} from 'react';
+import DialogRootContainer from '../../ui/DialogRootContainer.jsx';
+import {PopupPanel} from '../../ui/PopupPanel.jsx';
+import {primePlot} from '../PlotViewUtil.js';
+import {Tabs, Tab} from '../../ui/panel/TabPanel.jsx';
+import {callGetFitsHeaderInfo} from '../../rpc/PlotServicesJson.js';
+import {BasicTable} from '../../tables/ui/BasicTable.jsx';
+import {dispatchShowDialog, dispatchHideDialog} from '../../core/DialogCntlr.js';
+
+import HelpIcon from '../../ui/HelpIcon.jsx';
+
+import Band from '../Band.js';
+
+const popupIdRoot= 'fitsHeader';
+
+const popupPanelStyle = {
+	width: 450,
+	minWidth: 450,
+	height: 380,
+	minHeight: 300,
+	resize: 'both',
+	overflow: 'auto',
+	position: 'relative'
+};
+
+const tabStyle={width: '100%', height: '90%', overflow:'auto'};
+const tableStyle={width: '100%', height: '100%', overflow:'auto', flexGrow: 1, display: 'flex'};
+
+const titleStyle = {width: '100%', height: 20, whiteSpace: 'nowrap', display:'inline-block'};
+const closeButtonStyle = {'textAlign':'center', display:'inline-block', marginTop:10, width: '80%'};
+const helpIdStyle = {'textAlign':'center', display:'inline-block', marginTop:10, width: '20%'};
+const mTableStyle={width: '100%', height: 380, flexGrow: 1, display: 'flex'};
+
+
+const popupForm  =  (plot, fitsHeaderInfo,  popupId )=> {
```

Comment:
I don't think you should use the arrow syntax for a long functions.  Just use `function`